docs/tutorial/README.md

```diff
@@ -167,7 +167,7 @@ class Post extends Model
 {
     public function author(): BelongsTo
     {
-        return $this->belongsTo(User::class);
+        return $this->belongsTo(User::class, 'user_id');
```

Comment:
Let's change the migration to name the column `author_id` instead, that way we match the convention that Laravel expects.

Reply:
Done